### PR TITLE
Fix jax vit unit tests

### PR DESCRIPTION
**Pull request by bkowalskiINTEL**
This PR changes how the accuracy is evaluated in jax vit unit tests.
Previously: exact top-5 match with hardcoded labels
Now: exact top-1 match with label calculated on the original model + top-1 label propability error threshold

Also, now the input images for vit are in uint8 instead of float32

### CODE REVIEW

File: test/jax/test_vit_quantization.py [hunk 1]
```diff
@@ -30,23 +31,28 @@ def colva_beach_sq():
...
+    img = random.randint(key, shape=(1, 224, 224, 3), minval=0, maxval=256, dtype=jnp.uint8)
     return img
 
 
 def classify_image(model, image, labels_n=1):
     out = model(image)
     labels = decode_predictions(jnp.array(out), top=labels_n)[0]
-    return [class_name for (_, class_name, _) in labels]
+    probs = jax.nn.softmax(jnp.array(out)[0])
+    top_probs = [probs[i] for i in jnp.argsort(probs, descending=True)[:5]]
```
[the remark this answers is not in the document]
Comment (bkowalskiINTEL) [reply test/jax/test_vit_quantization.py]: Done

File: test/jax/test_vit_quantization.py [hunk 2]
```diff
@@ -24,29 +25,34 @@ def colva_beach_sq():
...
 
 
-def classify_image(model, image, labels_n=1):
+def classify_image(model, image, top_k=1):
     out = model(image)
-    labels = decode_predictions(jnp.array(out), top=labels_n)[0]
-    return [class_name for (_, class_name, _) in labels]
+    labels = decode_predictions(jnp.array(out), top=top_k)[0]
+    probs = jax.nn.softmax(jnp.array(out)[0])
+    top_probs = [probs[i] for i in jnp.argsort(probs, descending=True)[:top_k]]
```
[the remark this answers is not in the document]
Comment (bkowalskiINTEL) [reply test/jax/test_vit_quantization.py:42]: Not true: [https://docs.jax.dev/en/latest/_autosummary/jax.numpy.argsort.html](url)

File: test/jax/jax_test_utility.py [hunk 3]
```diff
@@ -101,17 +101,14 @@ def compute_expected_qdq_dense_output(
     return current_input, all_a_scales, all_w_scales
 
 
-def load_image(image_path, target_size, normalize):
+def load_image(image_path, target_size):
     with Image.open(image_path) as img:
         if img.mode != "RGB":
             img = img.convert("RGB")
         img = img.resize(target_size, Image.BILINEAR)
         pixels = jnp.array(img)
```
[the remark this answers is not in the document]
Comment (bkowalskiINTEL) [reply test/jax/jax_test_utility.py:109]: The only image we're using now - colva_beach_sq.jpg - loads in uint8, so we don't need to normalize it. I don't think we'll be adding another images, but if at some time we do, we'll take care of that then.

**Review by wpietka — APPROVED** [review]
LGTM

**Review by anko-intel — APPROVED** [review]
LGTM
